### ISSUE TITLE
Adding ability to make the results page be visible when the researcher desires it 

### ISSUE THREAD

**Issue by mschwamb** [zooniverse/Panoptes#2526]
I was told this was a panoptes issue (https://github.com/zooniverse/Panoptes-Front-End/issues/3864#issuecomment-343469387)

With the autosave feature if I start working on the results page it automatically gets made public and shows as I write it. It would be nice to be able to work on it without it being made public right away. Also if there's a press release, a researcher would want to have a results page likely but needs to make it visible when the embargo drop.s Maybe there could be a check box to make it linked to the navigation bar on the site and otherwise available but only by directly access the url?

I needed to have the results page live for the journal editor and copy editor to accept my paper proofs as I'm linking the dataset off of the results page. It would be really nice to not have to make a separate project to do the copy editing and formatting on it so that volunteers don't see a page in progress. Most other pages for a project you create while you're building the project before its launched. The results page is the only page that you write while the project is active, and if you have a press release you might want to not have it accessible from the menu bar until the embargo lifts.

**Comment by camallen** [comment]
Is the use case a long term embargo? E.g. make this page public on Feb 21st 2018? Or is it just autosave while you are typing?  

The former will require a notion of embargo on pages.

**Comment by mschwamb** [comment]
@camallen couldn't the researcher be the one decide to hit the publish button like a blog where until you hit publish it's a draft not live/visible on the site?

**Comment by camallen** [comment]
Yep - so the notion of published pages where they default to false until actively published in the builder.

**Comment by eatyourgreens** [comment]
Once `published` is changed from false to true, I assume edits to the page appear instantly, like now. 

Should the project builder also include an option to turn off the published flag? I'm thinking a checkbox, maybe. Unchecking it removes the page from the site.

**Comment by camallen** [comment]
Sounds ideal @eatyourgreens - something to follow up with once API support is in.

**Comment by eatyourgreens** [comment]
👍🏻 Sounds good.

If proper versioning is needed for the project web pages, I'd suggest that project owners build a small site using a small Content Management System like Wordpress. Something like http://junglerhythms.org/. That makes more sense to me than us trying to build our own CMS as part of Panoptes.

**Comment by mschwamb** [comment]
I think that's a great solution. It gives the ability to hide the results page until a press release comes out for example or if the researcher is working on it. I think it's fine to make the project owner manually set it to published and then the page behaves like every other.
